server/views/layout/default.njk

```diff
@@ -12,17 +12,17 @@
     <style>
       {% include "partials/critical.css.njk" %}
     </style>
-    <link rel="preload" href="/assets/css/{{ 'non-critical.css' | getHashedFile }}" as="style" onload="this.rel='stylesheet'">
+    <link rel="preload" href="{{ '/assets/css/' | getHashedFile('non-critical.css') }}" as="style" onload="this.rel='stylesheet'">
     <link rel="apple-touch-icon" sizes="180x180" href="/assets/icons/apple-touch-icon.png"/>
 
-    <link rel="shortcut icon" href="/assets/icons/favicon.ico" type="image/ico">
-    <link rel="icon" type="image/png" href="/assets/icons/favicon-32x32.png" sizes="32x32"/>
-    <link rel="icon" type="image/png" href="/assets/icons/favicon-16x16.png" sizes="16x16"/>
-    <link rel="manifest" href="/assets/icons/manifest.json"/>
-    <link rel="mask-icon" href="/assets/icons/safari-pinned-tab.svg" color="#000000"/>
-    <script src="/assets/libs/picturefill.min.js" async></script>
+    <link rel="shortcut icon" href="{{ '/assets/icons/' | getHashedFile('favicon.ico') }}" type="image/ico">
+    <link rel="icon" type="image/png" href="{{ '/assets/icons/' | getHashedFile('favicon-32x32.png') }}" sizes="32x32"/>
+    <link rel="icon" type="image/png" href="{{ '/assets/icons/' | getHashedFile('favicon-16x16.png') }}" sizes="16x16"/>
+    <link rel="manifest" href="{{ '/assets/icons/' | getHashedFile('manifest.json') }}'"/>
+    <link rel="mask-icon" href="{{ '/assets/icons/' | getHashedFile('safari-pinned-tab.svg') }}" color="#000000"/>
+    <script src="{{ '/assets/libs/' | getHashedFile('picturefill.min.js') }}" async></script>
```

Comment:
Not sure I like this as a function.
@wellcometrust/experience-dev Any idea?

Reply:
what are the other options? I think it looks OK.

Reply:
I haven't thought of any, if you're good, I'm 😆